backend/gitSound.py

```diff
@@ -191,28 +191,30 @@ def removeSongFromPlaylist(self, pid, songid):
         user = pidSplit[0]
         playId = pidSplit[1]
 
-        repo = pygit2.Repository(self.gitDir + user + "/" + playId)
+        self.repo = pygit2.Repository(self.gitDir + user + "/" + playId)
 
         # create the file blob
-        fileBlob = repo.create_blob_fromdisk(
+        fileBlob = self.repo.create_blob_fromdisk(
             self.gitDir + pid + "/index.txt")
 
-        newTree = repo.TreeBuilder()
+        newTree = self.repo.TreeBuilder()
 
         # insert it into the tree
         newTree.insert("index.txt", fileBlob,
                        os.stat(self.gitDir + pid + "/index.txt").st_mode)
 
-        tree = newTree.write()
+        self.tree = newTree.write()
 
-        # add it to the commit
-        repo.index.read()
-        repo.index.add("index.txt")
-        repo.index.write()
+    def commitChangesToPlaylist(self, pid, tree):
+        
+        # add to commit 
+        self.repo.index.read()
+        self.repo.index.add("index.txt")
+        self.repo.index.write()
 
-        # commit
-        repo.create_commit("HEAD", self.author, self.comitter,
-                           "Removed " + songid, tree, [repo.head.target])
+        # commit changes to playlist
+        self.repo.create_commit("HEAD", self.author, self.comitter,
+                           "Changes committed to " + pid, tree, [self.repo.head.target])
```

Comment:
I agree. I put it in there as a placeholder for now. If you think of something better, we should change it. Right now we are not keeping a list of all changes between commits, so there isn't a way to give feedback on all of the changes